app/models/vote.rb

```diff
@@ -82,5 +82,11 @@ def evaluate_reward(new_vote, value, subject)
       end
       reward
     end
+
+    def voted(resource, user)
+      vote = where(user_id: user.id, voteable_id: resource.id, voteable_type: resource.class.to_s).first
+      return vote.value if vote
+      0
```

Comment:
Unfortunately, you I can't do this, because this will give me the vote object as a hash, whereas I want just the `value` field. :disappointed: